kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -127,7 +128,7 @@ export default function useDownloadRequests(store) {
     const data = {
       contentnode_id: contentNode.id,
       metadata,
-      source_id: store.getters.currentUserId,
+      source_id: useUser().currentUserId,
```

Comment:
As this is being used outside of a component, this would need to be `useUser().currentUserId.value`